packages/lib-react-components/src/Modal/Modal.js

```diff
@@ -43,14 +46,20 @@ class Modal extends React.Component {
         elevation='xlarge'
         fill
         pad='none'
+        {...props}
       >
         <ModalHeading
           background={headingBackground}
           className={className}
           closeFn={closeFn}
+          color={titleColor}
           title={title}
         />
-        <ModalBody className={className} pad={pad}>
+        <ModalBody
+          className={className}
```

Comment:
I don't think there'd be an issue, but confirming there couldn't be a conflict or issue from ModalHeading and ModalBody sharing the same `className`?

Reply:
CSS classes can be reused, so I don't think so.